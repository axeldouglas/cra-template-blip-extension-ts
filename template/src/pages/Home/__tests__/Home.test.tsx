import { render } from '@testing-library/react';

import Home from '..';

describe('<Home />', () => {
    it('should render the Home', () => {
        const { container } = render(<Home />);

        expect(container.firstChild).toMatchSnapshot();
    });
});
