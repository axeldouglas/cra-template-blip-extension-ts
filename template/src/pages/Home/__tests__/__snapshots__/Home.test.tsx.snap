// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> should render the Home 1`] = `
<div
  class="flex flex-col justify-center items-center py-24 h-full text-neutral-dark-city"
>
  <img
    alt="BLiP"
    class="App-logo h-32"
    src="[object Object]"
  />
  <div
    class="mt-4"
  >
    <p
      class="text-center"
    >
      paragraph.homeDescription.part1
    </p>
    <p
      class="text-center"
    >
      Edit 
      <code>
        src/pages/Home.js
      </code>
       and save to reload.
    </p>
  </div>
</div>
`;
